lib/mayaUsd/render/vp2RenderDelegate/mesh.cpp

```diff
@@ -367,6 +429,324 @@ HdVP2Mesh::HdVP2Mesh(HdVP2RenderDelegate* delegate, const SdfPath& id, const Sdf
 #endif
 }
 
+void HdVP2Mesh::_CommitMVertexBuffer(MHWRender::MVertexBuffer* const buffer, void* bufferData) const
+{
+    const MString& rprimId = _rprimId;
+
+    _delegate->GetVP2ResourceRegistry().EnqueueCommit([buffer, bufferData, rprimId]() {
+        MProfilingScope profilingScope(
+            HdVP2RenderDelegate::sProfilerCategory,
+            MProfiler::kColorC_L2,
+            rprimId.asChar(),
+            "CommitBuffer"); // TODO: buffer usage so we know it is positions normals etc
```

Comment:
I commented about the swapping of the two string arguments in basisCurves.cpp. It is the same here.